liquibase-standard/src/main/java/liquibase/sqlgenerator/core/InsertGenerator.java

```diff
@@ -67,8 +68,10 @@ public void generateValues(StringBuilder sql,InsertStatement statement, Database
 
         for (String column : statement.getColumnValues().keySet()) {
             Object newValue = statement.getColumnValues().get(column);
-            if ((newValue == null) || "NULL".equalsIgnoreCase(newValue.toString())) {
+            if ((newValue == null) || newValue.toString() == null) {
```

Comment:
I think the second part of this IF will never be true, I think it would be just if `newValue` were of `Null` type.